### PR TITLE
Finish club detail page

### PR DESCRIPTION
**Pull request by wangwenyue**
Finish add money page.

Finish create club page.

Finish club detail page.

Add Modal for club detail page, add money page and create club page.

Write deploy instruction in README.md.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,49 @@
 # club-check-in
 Front-end of club-check-in
+
+
+## Deployment
+
+### Instructions
+
+0.Install all dependency
```
Comment (Keith-CY) [inline README.md]: dependency => dependencies

File: README.md [hunk 2]
```diff
@@ -1,2 +1,49 @@
...
+
+```shell
+yarn install
+```
+
+1.Use webpack generate production version
```
Comment (Keith-CY) [inline README.md]: Use webpack `to` generate

File: src/containers/Manage/ManageClubDetail/index.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        <Link to={'./modify'}>
+          <div className='manageClubDetail__container--content'>
+            <div className='manageClubDetail__content--name'>
+              Cryptape慢跑俱乐部
+            </div>
+            <div className='manageClubDetail__container--deposit'>
+              <div className='manageClubDetail__container--deposit-info'>
+                <span className='manageClubDetail__content--deposit'>社团经费：2000.00</span>
+                <div className='manageClubDetail__content--separate-line'></div>
+                <span className='manageClubDetail__content--id'>ID:1001</span>
```
Comment (Keith-CY) [inline src/containers/Manage/ManageClubDetail/index.js]: Normally we add space after `:`

File: src/containers/Manage/index.js [hunk 4]
```diff
@@ -1,12 +1,37 @@
...
     return (
-      <div>Club Management page</div>
+      <div className='manageClubDetail__container--club'>
+        <Header titleName='社团管理' backRoute='./detail'/>
+        <div className='manageClubDetail__container--club-info'>
+          <ManageClubDetail/>
+          <Flex className='manageClubDetail__container--increase-money' justify='center'>
+            <InputItem
+              className='manageClubDetail__input--increase-money'
+              placeholder="金额"
```
Comment (Keith-CY) [inline src/containers/Manage/index.js]: single quote

File: src/containers/New/index.js [hunk 5]
```diff
@@ -1,12 +1,46 @@
...
+          <TextareaItem
+            className='newClub__content--club-rule'
+            placeholder='有什么要嘱咐团员的都写在这里吧'
+            rows={8}
+            count={180}
+          />
+        </Flex>
+        <InputItem
+          className='newClub__input--club-deposit'
+          placeholder="写了你就别后悔"
```
Comment (Keith-CY) [inline src/containers/New/index.js]: single quote

File: src/containers/New/index.js [hunk 6]
```diff
@@ -1,12 +1,46 @@
...
+        >社团经费</InputItem>
+        <InputItem
+          className='newClub__input--report-threshold'
+          placeholder="看你心情咯"
```
Comment (Keith-CY) [inline src/containers/New/index.js]: single quote

**Review by Keith-CY — CHANGES_REQUESTED** [review]

**Comment by wangwenyue** [comment]
@Keith-CY all request resolved.

**Review by Keith-CY — APPROVED** [review]

**Pull request merged**